include/ignition/gui/qml/GzVector3.qml

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+
+/**
+ *  Item displaying a 3D vector
+ *
+ *  Users should load values to xValues, yValues, and zValues.
+ *  If readOnly == False,
+ *  users can read from signal parameters of GzVectorSet: _x, _y, and _z
+ *
+ *  Usage example:
+ *  GzVector3 {
+ *    id: gzVector
+ *    xName: "Red"
+ *    yName: "Green"
+ *    zName: "Blue"
+ *    gzUnit: ""
+ *    readOnly: false
+ *    xValue: xValueFromCPP
+ *    yValue: yValueFromCPP
+ *    zValue: zValueFromCPP
+ *    onGzVectorSet: {
+ *      myFunc(_x, _y, _z, _roll, _pitch, _yaw)
+ *    }
+ *  }
+**/
+Rectangle {
```

Comment:
The widget's background color is plain white, but it should follow the parent's color.

Before this PR:

![image](https://user-images.githubusercontent.com/5751272/178051615-47d2ad8f-3cf8-424e-8b10-b5f1bf23f24f.png)


With this PR:

![image](https://user-images.githubusercontent.com/5751272/178051496-1e8f7bf1-28c2-4467-8287-4e1df439fa9e.png)